src/oemof/network/energy_system.py

```diff
@@ -83,8 +84,8 @@ class EnergySystem:
     <oemof.core.network.Entity>` will always be grouped by their :attr:`uid
     <oemof.core.network.Entity.uid>`:
 
-    >>> from oemof.network import Entity
-    >>> from oemof.network import Bus, Sink
+    >>> from oemof.network.network import Entity
+    >>> from oemof.network.network import Bus, Sink
```

Comment:
Is that a remnant from the "nose" doctest import problem which should be fixed?
If not, I'll problaby rename `oemof.network.network` to something less redundant
and re-export the stuff from `oemof.network` via its "__init__.py".

Reply:
I'm not sure but feel free to change it.